src/Uno.UI/UI/Xaml/UIElement.TextHelper.wasm.cs

```diff
@@ -146,6 +147,21 @@ private void SetTextTrimming(object localValue)
 			}
 		}
 
+		internal void ApplyFlowDirection(bool isRtl)
+		{
+			// TODO: This will break if RenderTransform is used on the TextBlock.
+			// One of the transforms will override the other.
+			// TODO: Verify whether this breaks TextBlock if RTL FlowDirection is set "locally" (not inherited)
+			if (isRtl)
+			{
+				this.SetNativeTransform(new Matrix3x2(-1.0f, 0.0f, 0.0f, 1.0f, 0.0f, 0.0f));
```

Comment:
Won't this break text selection? Or at least make it weird?

Reply:
I'll need to verify